specs/GetSecurityDescriptorForAppContainerNames/GetSecurityDescriptorForAppContainerNames.md

```diff
@@ -61,6 +61,26 @@ STDAPI GetSecurityDescriptorForAppContainerNames(
 
 If the function succeds, the returned `SECURITY_DESCRIPTOR` must be freed by calling [LocalFree](https://docs.microsoft.com/en-us/windows/desktop/api/winbase/nf-winbase-localfree).
 
+## WinRT
+```c#
+namespace Microsoft.Security.AccessControl
+{
+  struct AppContainerAccess
```

Comment:
Update name to match native one, if necessary. 
Also, why struct? Win32 structs are extensible, but WinRT structs are not. Should this be a runtime class?

Reply:
RECOMMEND: Yes, rename to match the C++ one.

RECOMMEND: Switch to runtimeclass here for less breakage